src/Components/TextField/index.js

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from "react-i18next";
+
+const TextField = ({ name, onChange, value = "", title }) => {
+    const { t } = useTranslation();
+
+    return (
+        <label className="block text-gray-700 text-sm font-bold mb-5" htmlFor={name}>
+            {title && <p>{t(title)}</p>}
```

Comment:
Actually, we translate the TextField title by this line. Or it won't work with new changes with translating?

Reply:
Yes, it's fine for us to do the translation here, but the translation library can only add translations to the json files if a string literal is parsed into `t`, not a variable, so we need to add this dummy function.

You can see we did something similar [here](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/main/src/hooks/useCountryData.js#L7).

If you'd prefer, I can just add this myself and you can take a look after?

Reply:
Oh, I got this, thanks :)
I'll fix this, but should we then replace this:
````javascript
{title && <p>{t(title)}</p>}
````
to: 
````javascript
{title && <p>{title}</p>}
````
?